app/uk/gov/hmrc/merchandiseinbaggage/connectors/CurrencyConversionConnector.scala

```diff
@@ -27,13 +27,16 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class CurrencyConversionConnector @Inject() (
+  val appConfig: AppConfig,
   httpClient: HttpClient,
   @Named("currencyConversionBaseUrl") baseUrl: String
-) extends CurrencyConversionConfiguration {
+) {
 
   def getConversionRate(code: String, date: LocalDate = LocalDate.now())(implicit
     hc: HeaderCarrier,
     ec: ExecutionContext
   ): Future[Seq[ConversionRatePeriod]] =
-    httpClient.GET[Seq[ConversionRatePeriod]](s"$baseUrl${currencyConversionConf.currencyConversionUrl}$date?cc=$code")
+    httpClient.GET[Seq[ConversionRatePeriod]](
+      s"$baseUrl${appConfig.currencyConversionConf.currencyConversionUrl}$date?cc=$code"
```

Comment:
After cleanup, can update the url to `s"${appConfig.currencyConversionUrl}/currency-conversion/rates/$date?cc=$code"`, thanks